_sass/screen.scss

```diff
@@ -5,6 +5,18 @@ body {
   font-size: $base-font-size;
 }
 
+p {
+  max-width: 40em;
+}
+
+.usa-font-lead{
+  max-width: 20em;
```

Comment:
I think we should use a `ch` value here for character width or perhaps break the content with a `<br>` tag